src/components/section-display.yml

```diff
@@ -0,0 +1,7 @@
+sectionDisplay:
+    notSupportedSectionEmptyStateTitle:
+      en: This content can not be displayed
```

Comment:
I looked it up: "cannot" and "can not" have the same level of formality, the informal way for both is "can't". The recommended because usual way to write it is "cannot" (that's also what we use in some other places). So I'd suggest to remove the space.